packages/transitive-overlay/src/TransitiveOverlay.story.tsx

```diff
@@ -337,3 +337,26 @@ export const OTP2ScooterItinerary = injectIntl(({ intl }) => (
   </>
 ));
 OTP2ScooterItinerary.decorators = [withMap([33.749, -84.388], 11)];
+
+export const ShowRouteArrows = (): JSX.Element => (
+  <>
+    <TransitiveOverlay
+      showRouteArrows
+      transitiveData={itineraryToTransitive(bikeOnlyItinerary, { companies })}
```

Comment:
Can we use an itinerary with transit instead?

Reply:
Sure.